src/NUnitEngine/nunit.engine.tests/nunit.engine.tests.csproj

```diff
@@ -42,12 +42,6 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Mono.Cecil.0.9.6.1\lib\net20\Mono.Cecil.dll</HintPath>
     </Reference>
-    <Reference Include="Mono.Cecil.Mdb">
```

Comment:
These assemblies weren't in the NuGet package that I restored causing warnings. They don't seem to be used.